### PR TITLE
[13.0] [MIG] base report to printer migration

### PR DESCRIPTION
**Pull request by schout-it**
* removed api.multi's
* changed color to decorator in tree view
* javascript fail threw an error, fixed it.

### CODE REVIEW

**Comment by schout-it** [comment]
seems pycups is not installed from the manifest

**Comment by pedrobaeza** [comment]
Please add it in this PR in requirements.txt file in the root

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pycups==1.9.74
```
Comment (bealdav) [inline requirements.txt:1]: why fix static version and not a minimal version with >=

**Review by bealdav — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (schout-it) [reply requirements.txt:1]: I upgraded it from the last version (1.9.73 in v12)
It's tested and working with this version, not sure about future versions

File: base_report_to_printer/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Report to printer",
+    "version": "13.0.1.0.0",
+    "category": "Generic Modules/Base",
+    "author": "Agile Business Group & Domsense, Pegueroles SCP, NaN,"
+    " LasLabs, Camptocamp, Odoo Community Association (OCA),"
+    " Open for Small Business Ltd",
+    "website": "http://www.agilebg.com",
```
Comment (rven) [inline base_report_to_printer/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/report-print-send",
```

**Review by lembregtse — APPROVED** [review]

**Review by rven — CHANGES_REQUESTED** [review]

**Review by rven — APPROVED** [review]

**Pull request merged**